e2e/passcode/post-passcode-setup.yaml

```diff
@@ -0,0 +1,71 @@
+appId: com.comapeo
+env:
+  CAT: MAIN
+---
+- tapOn:
+    id: '${CAT}.drawer-icon'
+- tapOn: 'Security'
+- tapOn: 'App Passcode'
+- assertVisible:
+    label: 'From Security screen, [On click] App Passcode button prompts password entry'
+    text: 'Enter Passcode'
+- runFlow: './subflows/check-number-keyboard.yaml'
+- tapOn:
+    id: '${CAT}.header-back-btn'
+- assertVisible:
+    label: 'Back button returns to the Security Screen'
+    id: '${CAT}.security-scrn'
```

Comment:
nit: is there a way to assert to a specific part of the screen? for example, think it could be preferable to strictly check the navigation header and make sure it says `Security` to know that you're on the correct screen.

Reply:
seems like this could be possible using relative position selectors: https://maestro.mobile.dev/api-reference/selectors#relative-position-selectors

would require adding an selectable id to the navigation header i guess, if possible

Reply:
I tried to use those with the radio button and they didn't work, but I can try again

Reply:
I am not really sure why you wanted this since I put a test id on the one screen and that is all it could be (but maybe there will be more screens in the future?) and it is a little wonky, but if that is what you want... It is now done.

Reply:
the primary motivation is to have the e2e test emulate the user experience more closely. using this case as an example, the user can know that they're returning to the security screen because they see the `Security` text in the header after the interaction of going back. asserting that the navigation header has that text is more reflective of that experience than asserting the presence of an opaque code-only id.

there are definitely cases where this doesn't work or is difficult to do, but i think it's worth considering the approach for the cases where it could be done.